File: 02_algorithm/programmers/Level1/두 개 뽑아서 더하기.py
def solution(numbers):
    answer = []
    for i in range(len(numbers)):
        for j in range(i + 1, len(numbers)):
            if numbers[i] + numbers[j] not in answer:
                answer.append(numbers[i] + numbers[j])
    return sorted(answer)

print(solution([2,1,3,4,1]))

File: 06_js/04_algorithm/programmers/Level1/두 개 뽑아서 더하기.js
function solution(numbers) {
  const answer = new Set();
  const numbersCount = numbers.length;
  for (let i = 0; i < numbersCount; i++) {
    for (let j = i + 1; j < numbersCount; j++) {
      answer.add(numbers[i] + numbers[j]);
    }
  }
  return Array.from(answer).sort((a, b) => a - b);
}

console.log(solution([2, 1, 3, 4, 1]));
console.log(solution([5, 0, 2, 7]));